### ISSUE TITLE
Acessibility review

### ISSUE THREAD

**Issue by crowbot** [mysociety/alaveteli#3095]
Do an assessment of how Alaveteli shapes up on accessibility. Outcome would be a list of improvements, ideally in priority order. 

Possible resources:

- http://pa11y.org
- https://www.gov.uk/service-manual/user-centred-design/accessibility

**Comment by wrightmartin** [comment]
Here's a quick overview of the accessibility of the default alaveteli theme. Overall I found it to be quite good, with only very minor issues. Do you want these ticketed up individually?

- Missing form label on feedback box
- Hide js related controls (e.g. menu, close etc) from screenreader and non-js clients
- Search bar on authorities page is missing a label
- Add ARIA Landmark roles
- Add skip navigation & back to top links
- Missing form label on select_authority search
- Can’t activate language switcher with keyboard
- Numbers in the ‘how it works’ section should come before the text snippet, or not at all
- Contrast of the ‘status’ text on the request page is too low
- Orange colour is too low contrast for text
- White on green colour is too low contrast for text

**Comment by garethrees** [comment]
https://webmasters.googleblog.com/2011/09/pagination-with-relnext-and-relprev.html?m=1

**Comment by crowbot** [comment]
@wrightmartin - is there anything left now in this ticket that hasn't been implemented?

**Comment by wrightmartin** [comment]
@crowbot it's mostly finished, there are a few more things we could do - but they are more additions rather than fixes. As a PR of accessibility fixes, this is complete

**Comment by crowbot** [comment]
Thanks @wrightmartin - I'm going to close this ticket then. Is there a list of the improvements somewhere? Would be good to get that into a ticket.

**Comment by wrightmartin** [comment]
@crowbot just ticketing up a couple now
